addons/project/views/project_task_views.xml

```diff
@@ -504,7 +504,6 @@
                                placeholder="Private"
                                class="o_project_task_project_field"
                                domain="[('type_ids', 'in', context['default_stage_id'])] if context.get('default_stage_id') else []"
-                               context="{'default_type_ids': [(4, context.get('default_stage_id', False))]}"
```

Comment:
Is it really needed? 🧐

Reply:
No we don't need it as we are adding python code to set default stage
Don't you think? 🤔